src/assets/silent-check-sso.html

```diff
@@ -0,0 +1,7 @@
+<html>
+<body>
+<script>
+    parent.postMessage(location.href, location.origin);
```

Comment:
I know this is required by Keycloak, but can you explain to me what it actually does? :sweat_smile: